ansible/roles/ldap_server/tasks/tasks.yaml

```diff
@@ -188,7 +188,7 @@
       become: yes
       command: ldapmodify -Y EXTERNAL -H ldapi:/// -f /etc/ldap/certinfo.ldif
       register: ldapmodify
-      failed_when: ldapmodify.rc != 0 and "no equality matching rule" not in ldapmodify.stderr
```

Comment:
oh wow this is interesting. really cool that you can specify the failed_when conditions like this. Did you run into this error recently?

Reply:
Nevermind I see you said this in the commit. So my understanding is you get this error if you rerun it again (after all the values have already been modified). So the error is for a blank dif file. Again really new to this - sorry if this is a dump question.

Reply:
No, I don't often use the `failed_when` stuff. And yeah, was getting an issue running it now.